modules/ximgproc/test/test_find_ellipses.cpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "test_precomp.hpp"
+
+using namespace cv;
+
+namespace opencv_test { namespace {
+
+
+TEST(FindEllipsesTest, EllipsesOnly)
```

Comment:
In debug build mode there is catched out of boundary access issue:

```
[ RUN      ] FindEllipsesTest.EllipsesOnly
unknown file: Failure
C++ exception with description "OpenCV(4.6.0-dev) /build/precommit-contrib_linux64_no_opt/4.x/opencv/modules/core/include/opencv2/core/mat.inl.hpp:1603: error: (-215:Assertion failed) 0 <= y && y < size.p[0] in function 'operator[]'
" thrown in the test body.
[  FAILED  ] FindEllipsesTest.EllipsesOnly (18 ms)
```

@misakicoca Could have a chance to look on this?

Reply:
Ok, I have reproduced this issue locally so far and I will fix this bug in the following days.

Reply:
@alalek I have found the source of the error and found a way to fix it, how should I submit the code change?

Reply:
Thank you for looking on this!
Please open a new PR with the fix.

Reply:
Thanks very much for your advice! I have submitted the code changes in PR #3393.